lib/private/Config.php

```diff
@@ -290,6 +290,12 @@ private function writeData() {
 			throw new \Exception(sprintf('Could not acquire an exclusive lock on the config file %s', $this->configFilePath));
 		}
 
+		// Never write file back if disk space should be low (less than 10 MB)
+		$df = disk_free_space($this->configDir);
+		if ($df !== false && (int)$df < 10240) {
```

Comment:
```suggestion
		if ($df !== false && (int)$df < 10240000) {
```

The function returns free disk space in bytes.

Ref https://www.php.net/manual/en/function.disk-free-space.php#refsect1-function.disk-free-space-returnvalues

Reply:
ah you are right. I made a mistake in the comment. it should not be less than 10KB but we could think about increasing to 100KB. WDYT?

Reply:
fixed